### PR TITLE
[14.0][ADD] rma_scrap

### PR DESCRIPTION
**Pull request by DavidJForgeFlow**
This module allows you to put scrap the products.

@ForgeFlow

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
@DavidJForgeFlow please check my latest commit https://github.com/ForgeFlow/stock-rma/pull/305/commits/63c378fe61664cdd7905b8666abf798764777676

File: rma_scrap/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "RMA Put Away",
```
Comment (JordiBForgeFlow) [inline rma_scrap/__manifest__.py]: @DavidJForgeFlow Can you fix the name?

File: rma_scrap/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "version": "14.0.1.0.0",
+    "license": "LGPL-3",
+    "category": "RMA",
+    "summary": "Allows to put away the received products in odoo",
```
Comment (JordiBForgeFlow) [inline rma_scrap/__manifest__.py]: @DavidJForgeFlow same here

File: rma_scrap/__manifest__.py (hunk 1 above)
Comment (DavidJForgeFlow) [reply rma_scrap/__manifest__.py]: Done, thanks!

File: rma_scrap/__manifest__.py (hunk 2 above)
Comment (DavidJForgeFlow) [reply rma_scrap/__manifest__.py]: :+1:

**Pull request merged**